### PR TITLE
chore: upgrade svelte kit and migration work

### PR DESCRIPTION
**Pull request by ota-meshi**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -90,8 +90,8 @@
     "@fontsource/fira-mono": "^4.5.0",
     "@ota-meshi/eslint-plugin": "^0.12.0",
     "@sindresorhus/slugify": "^2.1.0",
-    "@sveltejs/adapter-static": "^1.0.0-next.26",
-    "@sveltejs/kit": "^1.0.0-next.406",
+    "@sveltejs/adapter-static": "^1.0.0-next.40",
+    "@sveltejs/kit": "^1.0.0-next.456",
```
Comment (JounQin) [inline package.json:94]: https://github.com/ota-meshi/eslint-plugin-svelte/pull/248

It seems kit has new breaking changes.

**Review by JounQin — CHANGES_REQUESTED** [review]

**Comment by baseballyama** [comment]
`prerender.default` also we need to remove.

File: package.json (hunk 1 above)
Comment (ota-meshi) [reply package.json:94]: I believe this PR addresses a breaking change in the kit. Were there other tasks that needed to be done?

Comment (JounQin) [reply package.json:94]: Is https://github.com/ota-meshi/eslint-plugin-svelte/pull/248#issuecomment-1236047343 resolved in this PR?

Comment (ota-meshi) [reply package.json:94]: If it was a codesandbox issue this PR should have fixed it. I am writing a comment in PR.

https://github.com/ota-meshi/eslint-plugin-svelte/pull/244/files#diff-bca34ff1efba1992c546adca0b1596996778b69817a7c391d29f40806694fdc9R2

**Review by JounQin — APPROVED** [review]

**Pull request merged**